Doc/howto/ipaddress.rst

```diff
@@ -32,7 +32,7 @@ A Note on IP Versions
 ---------------------
 
 For readers that aren't particularly familiar with IP addressing, it's
-important to know that the Internet Protocol is currently in the process
+important to know that the internet protocol is currently in the process
```

Comment:
I am not sure about this one.

Reply:
It is a proper name. Usually used as abbreviations IPv4 and IPv6. I think it should be titlecased.